File: __tests__/__fixtures__/output/node/axios/full.js
const axios = require("axios").default;
const { URLSearchParams } = require('url');

const encodedParams = new URLSearchParams();
encodedParams.set('foo', 'bar');

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value',
  headers: {
    cookie: 'foo=bar; bar=baz',
    accept: 'application/json',
    'content-type': 'application/x-www-form-urlencoded'
  },
  data: encodedParams,
};

axios.request(options).then(function (response) {
  console.log(response.data);
}).catch(function (error) {
  console.error(error);
});

File: src/targets/ruby/native/fixtures/full.rb
require 'uri'
require 'net/http'
require 'openssl'

url = URI("https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true

request = Net::HTTP::Post.new(url)
request["cookie"] = 'foo=bar; bar=baz'
request["accept"] = 'application/json'
request["content-type"] = 'application/x-www-form-urlencoded'
request.body = "foo=bar"

response = http.request(request)
puts response.read_body